src/common/const/nhost.js

```diff
@@ -1,5 +1,7 @@
-export const NHOST = {
-  AUTH_URL: `https://rgkjmwftqtbpayoyolwh.nhost.run/v1/auth/signin/provider/github?redirectTo=${encodeURI(
-    "http://localhost:3000/plays/create"
-  )}`,
+const AUTH_URL = (redirectURL, provider = "github") => {
+  return `https://rgkjmwftqtbpayoyolwh.nhost.run/v1/auth/signin/provider/${provider}?redirectTo=${encodeURI(
```

Comment:
The URL should be taken from .env file.

The .env already has the fragment of this URL to use. Do not create any new env variable.

Reply:
ah i was just tying with that and mistakenly pushed it also. fixing now.

Reply:
ah this was hardcoded from previous. our current prod has also this hardcoded btw.